learn/cookbooks/running_production.mdx

```diff
@@ -3,106 +3,106 @@ title: Running Meilisearch in production — Meilisearch documentation
-+ An ssh key pair to connect to that machine
+- A DigitalOcean droplet running Debian 10
+- An SSH key pair to connect to that machine
 
 <Capsule intent="tip">
-Learn how to connect via SSH to your [DigitalOcean droplet](https://www.digitalocean.com/docs/droplets/how-to/connect-with-ssh/) or any [Linux or Windows server](https://phoenixnap.com/kb/ssh-to-connect-to-remote-server-linux-or-windows)
+DigitalOcean has extensive documentation on [how to use SSH to connect to a droplet](https://www.digitalocean.com/docs/droplets/how-to/connect-with-ssh/).
 </Capsule>
 
 ## Step 1: Install Meilisearch
 
-To keep this tutorial as simple as possible, let's use a script that will carry out the installation process. It will copy a binary of Meilisearch to your machine and enable you to use it immediately.
+Log into your server via SSH, update the list of available packages, and install `curl`:
 
-Once you are logged in into your machine via SSH, ensure your system and its dependencies are up-to-date before proceeding with the installation.
-
-```bash
-# Update the list of available packages and their versions
+```sh
 apt update
-# Install curl which is required to install Meilisearch in the next step
 apt install curl -y
-# Install Meilisearch latest version from the script
-curl -L https://install.meilisearch.com | sh
 ```
 
-The different Meilisearch installation options are detailed in [this guide](/learn/getting_started/quick_start#setup-and-installation).
+Using the latest version of a package is good security practice, especially in production environments.
 
-**There are many different ways to get Meilisearch running on your machine.** As an open-source project, you can always compile the latest stable release of Meilisearch from its source code to ensure the binary uses your architecture in the best possible way.
+Next, use `curl` to download and run the Meilisearch command-line installer:
 
-You can always check the latest Meilisearch stable version, and get Meilisearch for the Operating System of your choice, by visiting the following link:
+```sh
+# Install Meilisearch latest version from the script
+curl -L https://install.meilisearch.com | sh
+```
 
-**[Latest Meilisearch Stable Version](https://github.com/meilisearch/meilisearch/releases/latest)**
+The Meilisearch installer is a set of scripts that ensure you will get the correct binary for your system.
 
-Give the binary execute permission using:
+Give the binary the required execute permission:
 
-```bash
+```sh
 chmod +x meilisearch
 ```
 
-Meilisearch is finally installed and ready to use. To make it accessible from everywhere in your system, move the binary file into your system binaries folder:
+The last step is to make the binary accessible from anywhere in your system. Move the binary file into `/usr/local/bin`:
 
-```bash
-# Move the Meilisearch binary to your system binaries
+```sh
 mv ./meilisearch /usr/local/bin/
 ```
 
+Meilisearch is now ready to use. 
```

Comment:
I think that's not true, right? Because you will recommend the user to configure the data and other stuff before...

Reply:
Hmmm…technically it is true because Meilisearch is indeed ready to use, but you're correct that it's sort of useless. I will update the text so it doesn't give the impression users have finished the tutorial when they reach this bit.